### PR TITLE
bug-fix/message-same-user-type-pop-up: Fixed bug for pop-up not appea…

### PR DESCRIPTION
**Pull request by viggy21**
Fixed an issue where the pop up wasn't appearing when the user was messaging another user of the same type

### CODE REVIEW

**Comment by kbib0002** [comment]
Please fix conflicts with **develop**

**Comment by kbib0002** [comment]
After testing, I have found that: after the user attempts to message another user of the same type, the pop-up will successfully appear and the user will be redirected to their messages page. However, if the user closes the pop-up it will reappear immediately - this loop continues indefinitely.

**Comment by kbib0002** [comment]
Please remove the console.log usages from lines 147 and 149. (??)

**Comment by viggy21** [comment]
Rebased with develop to resolve conflicts and console.log usages removed.

I just tried testing it and messaging a user of the same type - the pop-up appears and when it's closed, it stays closed and does not reopen for me so it's functioning as expected on my end.

**Comment by kbib0002** [comment]
Merge conflicts resolved and console.log usages removed. Thank you!

I am still getting that infinite loop unfortunately. I have deleted and re-pulled the entire codebase, and deleted and re-imported all of the mockdata. The account I created to showcase this was also new. You can see a recording attached to illustrate the issue.


https://github.com/user-attachments/assets/ee5491d1-a7d1-442d-93fc-cf9b1af074b8

File: behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx [hunk 1]
```diff
@@ -179,7 +181,7 @@ export const MessagesPage = () => {
             routeToChat(0);
         }
         // close the modal
-        onCloseModal();
+        //onCloseModal();
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx:184]: This might be causing the reloop not sure

Comment (viggy21) [reply behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx]: I found the fix for it!

**Review by kbib0002 — APPROVED** [review]
The infinite loop issue has been resolved.

**Pull request merged**